abis_mapping/templates/survey_site_data_v2/examples/minimal.ttl

```diff
@@ -81,6 +83,18 @@
     tern:featureType <http://linked.data.gov.au/def/tern-cv/5bf7ae21-a454-440b-bdd7-f2fe982d8de4> ;
     tern:locationDescription "Cowaramup Bay Road" .
 
+<http://createme.org/Site/P2> a tern:Site ;
+    void:inDataset <http://createme.org/dataset/Example-Systematic-Survey-Site-Dataset> ;
+    geo:hasGeometry _:N2d5e0873e8e72e68e4a9793300000001,
+        _:N53dd57c04e35e4aa3188c0b800000002 ;
+    schema:additionalType <http://linked.data.gov.au/def/tern-cv/8cadf069-01d7-4420-b454-cae37740c2a2> ;
+    schema:description "Fine woody debris." ;
+    schema:identifier "P2"^^<https://linked.data.gov.au/dataset/bdr/datatypes/siteID/WAM> ;
+    schema:name "Plot 2" ;
+    schema:sameAs <http://example.com/sites/S0> ;
```

Comment:
I think the ticket was asking for this to be 
```
schema:sameAs "http://example.com/sites/S0"^^xsd:anyURI
```
Or when the related site value is not a url, just a string literal
```
schema:sameAs "APASH02"
```

Reply:
scrapping this whole thing and starting again.